CONDUCT.md

```diff
@@ -0,0 +1,13 @@
+This document provides community guidelines for a safe, respectful, productive,
+and collaborative place for any person who is willing to contribute to the Opal
```

Comment:
@ylluminate You probably want to change the word "Opal" here to "V programming language" or something like that.

Reply:
Sorry, you're right. I just grabbed it real quick while I was thinking about it without modification, but yeah.